tests/core/test_policies.py

```diff
@@ -1229,3 +1231,34 @@ def test_get_training_trackers_for_policy(
 def test_deprecation_warnings_for_old_rule_like_policies(policy: Type[Policy]):
     with pytest.warns(FutureWarning):
         policy(None)
+
+
+class PolicyWithoutInitKwargs(Policy):
+    def __init__(self, *args):
+        pass
+
+    def persist(self, _) -> None:
+        pass
+
+    @classmethod
+    def _metadata_filename(cls):
+        return "no_finetune_policy"
+
+
+def test_loading_policy_with_no_constructor_kwargs(tmp_path: Path):
```

Comment:
Should we move these two tests closer to the similar tests for `load`?

Reply:
The other load tests are implemented on the test class so that it runs for each of the existing policies. Whereas these are a little different as they test the behaviour when using custom policy classes.

Reply:
ok